src/main.rs

```diff
@@ -85,6 +86,28 @@ async fn describe_instance(
         .await
 }
 
+/// An annotation attached to a pod by EKS describing the branch network interfaces when using per-pod security groups.
+/// example: [{"eniId":"eni-0e42914a33ee3c5ce","ifAddress":"0e:cb:3c:0d:97:3b","privateIp":"10.1.191.190","vlanId":1,"subnetCidr":"10.1.160.0/19"}]
```

Comment:
I wonder if this operator can race with EKS's ENI attachment... does it get re-invoked if the ENI configuration changes?

Reply:
If it fails to reconcile (ie: can't find the ENI ID), it will get re-queued and try again in 5 seconds.

Upon successful reconciliation, it will re-queue for 5 minutes in case something changed behind its back.